apps/web/tests/billing.spec.ts

```diff
@@ -12,6 +13,7 @@ test.describe('Billing', () => {
 
   test.beforeEach(async ({ page }) => {
     await initializeSession(page, { noTemplates: true });
+    await setFeatureFlag(page, FeatureFlagsKeysEnum.IS_IMPROVED_BILLING_ENABLED, true);
```

Comment:
We don't need 2 types of tests here based on FF - the number of trial days constant doesn't really matter - if it works for number N it will work in same way for every other